logging/conf/mimir.yml

```diff
@@ -0,0 +1,61 @@
+multitenancy_enabled: false
+
+activity_tracker: {}
+
+alertmanager: {}
+
+alertmanager_storage:
+  backend: local
+
+server:
+  http_listen_port: 9009
+
+  # Configure the server to allow messages up to 100MB.
+  grpc_server_max_recv_msg_size: 104857600
```

Comment:
we do not run an grpcs servers, why do we need them ?

Reply:
Mimir requires it for inter-communication between our mimir instance and the one that will be in grafana cloud as a backup cluster.